remote_report_to_printer/README.rst

```diff
@@ -0,0 +1,62 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================
+Report To Printer
+=================
+
+This module allows users to send reports to a printer using remote's configuration.
```

Comment:
I think you should explain what is a `remote` means.

Reply:
The information has been expanded.